External/YAeHMOP/EHTTools.h

```diff
@@ -24,11 +24,14 @@ struct RDKIT_EHTLIB_EXPORT EHTResults {
   unsigned int numAtoms;
   unsigned int numOrbitals;
   unsigned int numElectrons;
+  std::unique_ptr<double[]> overlapMatrix;
```

Comment:
I would just go to std::vector<double> here.  But I can see keeping the same coding style.